### PR TITLE
Updating template readme

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
-## Microsoft Open Source Code of Conduct
```
Comment (BillWagner) [inline README.md]: Should this have its own CoC file that references the .NET one? (That can be another PR, but it should be done).

**Comment by mairaw** [comment]
Good point @BillWagner. I'll add to the one I copied this from too. 😄
